spec/lib/google_maps_geocoder_spec.rb

```diff
@@ -83,23 +83,16 @@
   end
 
   context 'with google returns empty results' do
-    let(:empty_result) { { 'results' => [], 'status' => 'ZERO_RESULTS' } }
-    let(:limit_result) { { 'results' => [], 'status' => 'OVER_QUERY_LIMIT' } }
+    let(:results_hash) { { 'results' => [] } }
 
-    it 'raises an error if we have empty results' do
-      allow_any_instance_of(GoogleMapsGeocoder).to receive(:json_from_url)\
-        .and_return empty_result
+    GoogleMapsGeocoder::ERROR_STATUSES.each do |key, value|
```

Comment:
not really sure is it worth or not to make this in a tests, maybe I just need to stay with 5-times repeating code, since it might be easy to read, so if you think it's too much, I can fix it